### ISSUE TITLE
Function filter_author_archive_title() breaks get_the_author_archive() for other types of archives

### ISSUE THREAD

**Issue by stoniestbirch** [Automattic/Co-Authors-Plus#570]
Function filter_author_archive_title() breaks get_the_author_archive() for other types of archives (empty string returned).

See: #441 

The code now:
```php
	public function filter_author_archive_title() {
		if ( is_author() ) {
			$author = sanitize_user( get_query_var( 'author_name' ) );
			return "Author: ". $author;
		}
	}
```

What should be:
```php
 public function filter_author_archive_title($title) {
         if ( is_author() ) {
             $author = sanitize_user( get_query_var( 'author_name' ) );
             /* translators: Author archive title. 1: Author name */
             $title = sprintf( __( 'Author: %s' ), '<span class="vcard">' . $author . '</span>' );
         }
         return $title;
     }
```

**Comment by TheCrowned** [comment]
Hi Daniel,
thanks for reporting this! The issue was already fixed in #535 - it has not yet been released as a public update on WordPress.org, but the master branch of this repository contains the fix :)

**Comment by rebeccahum** [comment]
Closed as per ^^.
